api/inc/page_allocator_exports.h

```diff
@@ -45,7 +55,7 @@
 typedef struct {
     size_t page_size;       /* the page size in bytes */
     size_t page_count;      /* the number of pages in the page table */
-    void* page_origins[1];  /* table of pointer to the origins of a page */
+    void *page_origins[1];  /* table of pointer to the origins of a page */
```

Comment:
Can this go directly [here](https://github.com/ARMmbed/uvisor/pull/223/commits/bf52c4f1d702b9cb3afdae542a5a35bce519f3a9#diff-c1b58e0b36c6333e689bf1dae14a5044R48)?